<launch>

	<node pkg="robocup_nocom_pila" type="nodo_cml" name="cml" output="screen">
		<rosparam command="load" file="$(find robocup_nocom_pila)/config/cml.yaml"/>
	</node>
</launch>
